frontend/public/components/image-stream-tag.tsx

```diff
@@ -141,6 +148,39 @@ export const ImageStreamTagsDetails: React.SFC<ImageStreamTagsDetailsProps> = ({
           </div>
         )}
       </div>
+      <div className="co-m-pane__body-group">
+        <SectionHeading text={t('public~Supported Platforms')} />
+        {_.isEmpty(sortedSubmanifests) && importPolicyPreserveOriginal ? (
+          <span className="text-muted">{t('public~No Supported Platforms')}</span>
```

Comment:
Please use the `EmptyBox` component here

Reply:
Will do - thanks

Reply:
Resolved locally - it is in my updated commit.